Hotels.html

```diff
@@ -0,0 +1,250 @@
+                            </label>
+                        </li>
+                    </ul>
+                </fieldset>
+                <fieldset class="catalog-filter-group">
+                    <legend class="catalog-filter-title">Тип жилья:</legend>
+                    <ul class="catalog-filter-list">
+                        <li class="catalog-filter-item">
+                            <label class="control">
+                                <input class="control-input" type="radio" name="Гостиница" checked>
+
+                                Гостиница
+                            </label>
+                        </li>
+                        <li class="catalog-filter-item">
+                            <label class="control">
+                                <input class="control-input" type="radio" name="motel">
+
+                                Мотель
+                            </label>
+                        </li>
+                        <li class="catalog-filter-item">
+                            <label class="control">
+                                <input class="control-input" type="radio" name="apartments">
+
+                                Апартаменты
+                            </label>
+                        </li>
+                    </ul>
+                </fieldset>
+
+                <button class="button" type="submit">Применить</button>
+                <button class="button" type="reset">Сбросить</button>
+            </form>
+            <h2 class="catalog-filter-title">Найдено гостиниц: <span class="catalog-filter-title-number">38</span></h2>
+
+            <select class="select-control" aria-label="Сортировка">
+                <option value="popular">Сначала популярные</option>
+                <option value="cheap">Сначала дешёвые</option>
+                <option value="expensive">Сначала дорогие</option>
+            </select>
+            <a class="button button-light" href="#">
+                <span class="visually-hidden">Показать карточки</span>
+            </a>
+            <a class="button button-light" href="#">
+                <span class="visually-hidden">Показать слайды</span>
+            </a>
+            <a class="button button-light" href="#">
+                <span class="visually-hidden">Показать список</span>
+            </a>
+            <ul class="hotels-list">
+                <li class="hotels-card">
+                    <a href="hotels-card-link">
+                        <img class="hotels-card-image" src="#" alt="Описание картинки">
+                        <h3 class="hotels-card-title">Amara Resort & Spa</h3>
+                    </a>
+                    <p class="hotels-card-text">Гостиница</p>
+                    <p class="hotels-card-price">От 4000 ₽</p>
+                    <a class="hotels-card-button button" href="#">Подробнее</a>
+                    <button class="hotels-card-button button" type="submit">В избранное</button>
```

Comment:
```suggestion
                    <button class="hotels-card-button button" type="button">В избранное</button>
```
Так как тип `submit` используется только для кнопки внутри формы. Такая кнопка отвечает за отправку данных и в форме может быть только одна.

Для остальных кнопок в карточках гостиниц тоже исправьте.